### PR TITLE
test(animations): Fix e2e intermittent failures

### PR DESCRIPTION
**Pull request by jessicajaniuk**
This addresses a problem with animations e2e tests that were failing intermittently due to a missing timeout value

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [x] Other... Please describe: fixes intermittent CI failures

## What is the current behavior?

CI e2e tests fail intermittently

Issue Number: N/A

## What is the new behavior?
CI should consistently pass for e2e tests

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by dario-piotrowicz — APPROVED** [review]

File: aio/content/examples/animations/e2e/src/app.e2e-spec.ts [hunk 1]
```diff
@@ -169,7 +169,7 @@ describe('Animation Tests', () => {
   describe('Auto Calculation Component', () => {
     beforeAll(async () => {
       await autoHref.click();
-      await sleepFor(0);
+      await sleepFor(300);
```
Comment (AndrewKushnir) [inline aio/content/examples/animations/e2e/src/app.e2e-spec.ts:172]: I noticed the following code below in this spec file:
```
    const queryingAnimationDuration = 2500;

    beforeAll(async () => {
      await queryingHref.click();
      await sleepFor(queryingAnimationDuration);
    });
```

May be we could use similar format as a way to document the code (i.e. **why** we have 300 ms timeout here)?

Also, there is the `'Filter/Stagger Component'` block with a similar code, but with no timeout. Would it make sense to add it there too (with the right delay value)?

(there is also the `await sleepFor(400);` case in one of the suites in this file, so we can unify this code across the file by using the same structure)

**Review by AndrewKushnir — APPROVED** [review]
@jessicajaniuk looks great, thanks for fixing this! Just left a comment with a minor refactoring proposal.

File: aio/content/examples/animations/e2e/src/app.e2e-spec.ts (hunk 1 above)
Comment (jessicajaniuk) [reply aio/content/examples/animations/e2e/src/app.e2e-spec.ts:172]: Yeah I'm down for this post refactor. Should be pretty quick to do and would simplify things for future tests.

**Comment by alxhub** [comment]
This PR was merged into the repository by commit 787eaa7a4ba34d60c0516ffe13a2926bc673edce.

**Pull request closed without merging**